src/Controls/src/Core/Page/Page.cs

```diff
@@ -304,16 +403,26 @@ protected virtual void LayoutChildren(double x, double y, double width, double h
 			}
 		}
 
+		/// <summary>
+		/// When overridden, allows application developers to customize behavior immediately prior to the page becoming visible.
+		/// </summary>
 		protected virtual void OnAppearing()
 		{
 		}
 
+		/// <summary>
+		/// Application developers can override this method to provide behavior when the back button is pressed.
```

Comment:
This summary sounds more akin to a remark. I would perhaps change it be something simpler like "Determines the behavior when the back button of the page is pressed" and then move the current summary to a remark